docs/topics/api/addons.rst

```diff
@@ -284,6 +284,67 @@ This endpoint allows you to fetch a single version belonging to a specific add-o
     :>json string version: The version number string for the version.
 
 
+-----------------
+Review Notes List
+-----------------
+
+.. _review-notes-all:
+
+This endpoint allows you to list all the approval/rejection review history for a specific add-on, per version.
+
+.. http:get:: /api/v3/addons/addon/(int:addon_id|string:addon_slug|string:addon_guid)/reviewnotes/
```

Comment:
I'm not sure it should live under `/addons/addon/` as it's fairly specific. Thoughts ?